Automattic/sensei
# Quiz submission validation CLI command

Resolves #7077 

## Proposed Changes
* Add quiz submission validation CLI command.

## Testing Instructions
0. Make sure you have custom tables created. (Remove `sensei-version` from `wp_options` if you don't.)
1. Run `wp sensei validate quiz-submission` before running migration. It is expected to get a bunch of errors like `Tables-based submission not found`.
2. Run migration from Sensei LMS > Tools
If it is not the first run, delete two options:
```
DELETE FROM wp_options where option_name = 'sensei_migrated_progress_last_comment_id';
DELETE FROM wp_options where option_name = 'sensei_migrated_quiz_last_comment_id';
```
3. Run `wp sensei validate quiz-submission` after migration. No errors expected.
4. Now try to play with data. You can first remove some tables-based submissions, grades, and answers. After running the tool it is expected to see corresponding errors.
5. Now do the same with comments-based submissions, grades, and answers.

## Pre-Merge Checklist
